### PR TITLE
Update 00-spectro-k8s-dashboard.md

### PR DESCRIPTION
**Pull request by arvind-sys**
This PR adds new content to the Kubernetes Dashboard Pack related to RBAC. 
💻 [Preview](https://deploy-preview-1161--docs-spectrocloud.netlify.app/)
 * [Kubernetes Dashboard Pack](https://deploy-preview-1161--docs-spectrocloud.netlify.app/integrations/spectro-k8s-dashboard)
 * [Cluster Management RBAC]( https://deploy-preview-1161--docs-spectrocloud.netlify.app/clusters/cluster-management/cluster-rbac/#userbacwithoidc)

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1161#" title="Author self-approved">arvind-sys</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
Hi @arvind-sys. Thanks for your PR.

I'm waiting for a [spectrocloud](https://github.com/orgs/spectrocloud/people) member to verify that this patch is reasonable to test. If it is, they should reply with `/ok-to-test` on its own line. Until that is done, I will not automatically test new commits in this PR, but the usual testing commands by org members will still work. Regular contributors should [join the org](https://github.com/orgs/spectrocloud/people) to skip this step.

Once the patch is verified, the new status will be reflected by the `ok-to-test` label.

I understand the commands that are listed [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md
Comment (ritawatson) [inline content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md:39]:
```suggestion
- **Cluster Role** An entity that is assigned a set of access permissions scoped to the cluster and all of its Kubernetes namespaces. ClusterRoles do not have a namespace assigned.
```

File: content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md
Comment (ritawatson) [inline content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md:93]:
```suggestion
Use the steps below to create a RoleBinding or ClusterRoleBinding for your host clusters.
```

File: content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md
Comment (ritawatson) [inline content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md:100]:
```suggestion
Palette offers a set of [default roles](/user-management/palette-rbac#palettespecific(default)roles:) you can assign to your users. The Palette roles are only in scope at the platform level. This means you can manage the permissions for users' actions in Palette, such as creating or deleting clusters, creating projects, creating users, and more.
```

File: content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md
Comment (ritawatson) [inline content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md:137]:
```suggestion
* **Namespaces**: Use this tab to create a RoleBinding within Kubernetes namespaces.
```

File: content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md
Comment (ritawatson) [inline content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md:136]:
```suggestion
* **Cluster**: Use this tab to create a ClusterRoleBinds.
```

File: content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md
Comment (ritawatson) [inline content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md:147]:
```suggestion
2. Click on **Add New Binding**.
```

File: content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md
Comment (ritawatson) [inline content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md:151]:
```suggestion
  * Role Name: Define a custom role name to identify the cluster role.
```

File: content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md
Comment (ritawatson) [inline content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md:173]:
```suggestion
2. Click on **Add New Binding**.
```

File: content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md
Comment (ritawatson) [inline content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md:176]:
```suggestion
3. Add the namespace name or provide a regular expression to automatically apply the following settings to other namespaces in the future. Example: `/^web-app/`. Click on **Add To List**.
```

File: content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md
Comment (ritawatson) [inline content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md:182]:
```suggestion
5. Click on **Add New Binding**.
```

File: content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md
Comment (ritawatson) [inline content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md:185]:
```suggestion
6. Fill out the following details:
```

File: content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md
Comment (ritawatson) [inline content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md:188]:
```suggestion
  * Role Name: Define a custom role name to identify the cluster role.
```

File: content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md
Comment (ritawatson) [inline content/docs/04-clusters/06-cluster-management/09-cluster-rbac.md:240]:
```suggestion
This section explains how to use RBAC with OIDC providers. You can apply these steps to all the public cloud providers except [Azure-AKS](/clusters/public-cloud/azure/aks/#configureanazureactivedirectory) and [EKS](/integrations/oidc-eks/) clusters. Azure AKS and AWS EKS require different configurations. 
```

File: content/docs/06-integrations/00-kubernetes-dashboard.md
Comment (ritawatson) [inline content/docs/06-integrations/00-kubernetes-dashboard.md:31]:
```suggestion
A user-friendly version of this pack is available, Spectro Kubernetes Dashboard, which requires minimal configuration when used with the default settings. To learn more, check out [Spectro Kubernetes Dashboard](/integrations/spectro-k8s-dashboard).

```

File: content/docs/06-integrations/00-spectro-k8s-dashboard.md
Comment (ritawatson) [inline content/docs/06-integrations/00-spectro-k8s-dashboard.md:101]:
```suggestion
All IDP options below require you to map a set of users or groups to a Kubernetes RBAC role. There are two options you can use to get started with the Kubernetes Dashboard and an IDP.
```

**Review by ritawatson — APPROVED** [review]
The added flow diagram is helpful. 

I noticed that sometimes roleBinding is used and sometimes RoleBinding. I started correcting this to be RoleBinding and thought instead I'd just point it out. You may want to search and correct for consistency. 

Other minor corrections are needed.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1161#" title="Author self-approved">arvind-sys</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1161#pullrequestreview-1334065763" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by karl-cardenas-coding — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1161#" title="Author self-approved">arvind-sys</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1161#pullrequestreview-1335401084" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1161#pullrequestreview-1334065763" title="Approved">ritawatson</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
@arvind-sys: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | 2de5d6e1638a743c0b5d1c9b3b442def7a3aa611 | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1161/librarium_build/1634259923464359936) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Pull request merged**
